app/src/main/java/de/tum/in/tumcampusapp/component/other/notifications/model/AppNotification.kt

```diff
@@ -0,0 +1,25 @@
+package de.tum.`in`.tumcampusapp.component.other.notifications.model
+
+import android.app.Notification
+
+/**
+ * Holds a [Notification] that is to be displayed to the user. Its subclasses can be used to
+ * immediately display notifications via [InstantNotification] or to schedule notifications for
+ * later via [FutureNotification].
+ *
+ * @param id The identifier of the notification
+ * @param notification The [Notification] that will be displayed to the user
+ */
+abstract class AppNotification(val id: Int, val notification: Notification) {
+
+    companion object {
+
+        @JvmField val CAFETERIA_ID = 0
+        @JvmField val CALENDAR_ID = 1
+        @JvmField val NEWS_ID = 2
+        @JvmField val TRANSPORT_ID = 3
+        @JvmField val TUITION_FEES_ID = 4
```

Comment:
Uhhhh. Don't do this.
Subclasses are probably the best fit for this, but at least use something like [@IntDef](https://developer.android.com/reference/android/support/annotation/IntDef) or an enum for this purpose

Reply:
@thellmund probably a combination of subclasses and IntDef is best, so IDs can't clash :)